pandas/core/frame.py

```diff
@@ -1613,7 +1613,7 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
                  index=True, na_rep='NaN', formatters=None, float_format=None,
                  sparsify=None, index_names=True, bold_rows=True,
                  column_format=None, longtable=None, escape=None,
-                 encoding=None, decimal='.'):
+                 encoding=None, decimal='.',multicol=None,multirow=None):
```

Comment:
You'll need spaces after the `,`s here (check `git diff upstream/master | flake8 --diff`)